src/opnsense/mvc/app/library/OPNsense/Auth/Voucher.php

```diff
@@ -134,6 +136,7 @@ private function userNameExists($username)
 
     private function setStartTime($username, $starttime)
     {
+        $this->dbHandle->exec("PRAGMA journal_mode=off");
```

Comment:
I see, this needs more thinking. we might need to move the file away into it's own directory, but that has some migration challenges.